ui/src/routes/organizations/$orgId/products/$productId/repositories/$repoId/-types.ts

```diff
@@ -146,16 +146,16 @@ export type PackageManager = (typeof packageManagers)[number]['id'];
 
 export const reportFormats = [
   {
-    id: 'CycloneDx',
-    label: 'CycloneDX SBOM',
+    id: 'CycloneDX',
```

Comment:
IMO this line should be squashed into the commit that deals with the CycloneDX renaming to align with the plugin ID.